src/System.Linq/src/System/Linq/DebugView.cs

```diff
@@ -97,4 +97,34 @@ public object[] Items
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
         private IEnumerable _enumerable;
     }
+
+    internal sealed class GroupingDebuggerProxy<TKey, TElement>
+    {
+        private readonly Grouping<TKey, TElement> _grouping;
+
+        public GroupingDebuggerProxy(Grouping<TKey, TElement> grouping)
+        {
+            _grouping = grouping;
+        }
+
+        public TKey Key => _grouping.Key;
+
+        // The name of this property must alphabetically follow `Key` so the elements appear last in the display.
+        [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
+        public TElement[] Values => _grouping._elements;
```

Comment:
Self-note: Just realized this is incorrect bc the array can have trailing nulls/defaults. The correct approach would be to call `ToArray()` and cache it